### DISCUSSION TITLE
Two Panel Installation

### DISCUSSION THREAD

**Discussion by Caboose522** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#171] [Q&A] [answered]
I've seen a few examples of people mounting their monitoring boards in a separate enclosure next to their breaker panel, but not a lot of wiring diagrams for their full setup. I am in the US (200A residential split single phase) and want to put a monitor between my main 200A panel and a 100A sub-panel to measure the power for some circuits in both. How do people secure the IC wiring to the cabinet? Is it safe to run the ICs into two separate panels, or should I get a separate board/ESP32 for the other panel?

**Comment by CircuitSetup** [comment, marked as answer]
Do you mean the Current Transformer (CT) wiring? Most people that do this have to run extensions on the CTs. They can just be regular audio extension cables with 1/4" jacks. If you need to do this I would recommend calibrating the CT with the extension plugged in, as it will introduce a bit of resistance. 

As far as securing the wiring, some people run conduit between their panel and a separate box for the meter. If you're unsure how to do this I would highly recommend hiring an electrician.

You can use the same meter between both boxes. Just pay attention to what phase the AC transformer is plugged in to. This will determine the direction of the CTs on the wires. The will be opposite from each other, [like in this pic.](https://github.com/CircuitSetup/Split-Single-Phase-Energy-Meter/raw/master/images/current_transformer_SCT016-install2.jpg) If you get negative values as the output, this will indicated that the CT needs to be flipped on the wire.

**Reply by Caboose522** [reply]
Thank you for answering. Yes, the current transformer wiring needs to go into two separate panels. I will create a separate box with conduit between them then.

Just got the setup working yesterday, looking forward to getting a couple expansion boards to finish it out.
